Automattic/wp-calypso
# Proposal for stat type naming

Fixes: https://github.com/Automattic/dotcom-forge/issues/4962

## Proposed Changes

* Define Stat Types and granular Sub Types in a constants file
* To use less hardcoded strings and have a single place to check the stat naming

## Testing Instructions



* Make sure the DownloadCSV is still working with the testing instructions in https://github.com/Automattic/wp-calypso/pull/85190

## Pre-merge Checklist
